code/game/objects/items/devices/radio/radio.dm

```diff
@@ -1,11 +1,11 @@
 #define FREQ_LISTENING (1<<0)
 
-/obj/item/radio
+/obj/item/radio_abstract
```

Comment:
At this point it might as well be a component, is there anything in particular that it needs from being an item?
Also what difference does changing the typepath from 1 do? Is it just to indicate that this base typepath shouldn't be spawned in game?